src/fetchWrapper.js

```diff
@@ -52,11 +53,16 @@ const fetchWrapper = (url = '', options = {}) => {
         })
         .catch(error => {
             if (response) {
-                throw new ApiError(`Request failed with status ${ response.status }.`, error, response.status);
+                throw new apiError(`Request failed with status ${ response.status }.`, error, response.status);
             } else {
-                throw new ApiError(error.toString(), null, 'REQUEST_FAILED');
+                throw new apiError(error.toString(), null, 'REQUEST_FAILED');
```

Comment:
Why is this level of response-handling being done in the wrapper? Doesn't this mean that the library misses out on this functionality when run in-browser?

Reply:
I think the intention was to provide more flexibility for handling different error states down the line.
As far as browser support is concerned, I think there are several more hurdles that we have to overcome. I know we generally want to avoid introducing tooling where possible, but I think we've gotten to the point where it's worth it if we want to ensure compatibility.